src/reporter.h

```diff
@@ -31,12 +31,12 @@
 namespace google {
 
 // Configuration object.
-class MetadataAgentConfiguration;
+class Configuration;
 
 // A periodic reporter of metadata to Stackdriver.
 class MetadataReporter {
  public:
-  MetadataReporter(const MetadataAgentConfiguration& config,
+  MetadataReporter(const Configuration& config,
                    MetadataStore* store, double period_s);
```

Comment:
Same comment as in `updater.h`:
`MetadataStore* store,` can fit on the previous line now.

Reply:
Fixed